ui/src/services/api-config.service.tsx

```diff
@@ -0,0 +1,3 @@
+export default class ApiConfig {
```

Comment:
this is only a ts file since you aren't returning jsx. I would also continue with the naming notation above either - or ApiConfigService